mod/search/tests/phpunit/unit/Elgg/Search/SearchPluginTest.php

```diff
@@ -35,17 +35,15 @@ public function testParameterInitialization() {
 
 		$this->assertEquals([
 			'query' => 'literal needle',
-			'query_parts' => ['literal needle'],

```

Comment:
can not test this as it is not part of the plugin... assuming the provider will create them is incorrect

Reply:
It is in the search plugin parameter normalization

Reply:
query parts is generated by the service, not by the search plugin

Reply:
search plugin checks if the search string is quoted and sets to tokenization to false

Reply:
based on query, not on query_parts... i only removed the query_parts... search plugin does not know what the effect is of tokenization